ui/src/app/tasks/task-definitions/task-definitions.component.html

```diff
@@ -111,6 +107,15 @@ <h1 class="no-user-selection">Tasks</h1>
                       class="btn btn-default" style="margin-left: 0;" title="Launch">
                 <span class="glyphicon glyphicon-play"></span>
               </button>
+              <button *ngIf="schedulerEnabled" type="button" name="task-schedule{{ i }}" (click)="schedule(item)" [appRoles]="['ROLE_CREATE']"
+                      class="btn btn-default" style="margin-left: 0;" title="Schedule">
+                <span class="glyphicon glyphicon-time"></span>
+              </button>
+              <button *ngIf="schedulerEnabled" type="button" name="task-schedule-destroy{{ i }}" (click)="destroySchedules(item)"
+                      [appRoles]="['ROLE_CREATE']" class="btn btn-ico-remove-schedules btn-default" style="margin-left: 0;" title="Schedule">
+                <span class="glyphicon glyphicon-time"></span>
+                <span class="glyphicon glyphicon-remove"></span>
+              </button>
               <button name="task-destroy{{ i }}" type="button" [appRoles]="['ROLE_CREATE']" (click)="destroy(item)"
                       class="btn btn-default" style="margin-left: 0;" title="Destroy">
```

Comment:
Since we now have a button to remove Task Schedules, we should clarify this and maybe rename to "Destroy Task Definition"